src/ipc/ipc4/handler.c

```diff
@@ -510,13 +510,12 @@ static void ipc_compound_msg_done(uint32_t msg_id, int error)
 	}
 }
 
-/* wait for IPCs to complete on other cores and be nice to any LL work */
 static int ipc_wait_for_compound_msg(void)
 {
-	int try_count = 30; /* timeout out is 30 x 10ms so 300ms for IPC */
+	int try_count = 30;
 
 	while (atomic_read(&msg_data.delayed_reply)) {
-		k_sleep(Z_TIMEOUT_MS(10));
+		k_sleep(Z_TIMEOUT_US(250));
```

Comment:
I am not going to block PR because of this but can we change Z_TIMEOUT_US() to K_USEC()?
This expand to the same, but Z_xxxx definitions, I guess are Zephyr internal.